src/app/components/Transactions/ConsensusTransactions.tsx

```diff
@@ -59,27 +72,64 @@ export const ConsensusTransactions: FC<ConsensusTransactionsProps> = ({
           key: 'hash',
         },
         {
+          content: <BlockLink scope={transaction} height={transaction.block} />,
+          key: 'round',
+        },
+        {
+          align: TableCellAlign.Right,
           content: <Age sinceTimestamp={transaction.timestamp} />,
           key: 'timestamp',
         },
         {
           content: <ConsensusTransactionMethod method={transaction.method} truncate />,
-          key: 'method',
-        },
-
-        {
-          content: <ConsensusTransactionDetails ownAddress={ownAddress} transaction={transaction} />,
-          key: 'details',
+          key: 'type',
         },
         {
           align: TableCellAlign.Right,
           content: (
-            <RoundedBalance
-              value={transaction.body?.amount || transaction.body?.amount_change}
-              ticker={transaction.ticker}
-            />
+            <Box
+              sx={{
+                display: 'flex',
+                alignItems: 'center',
+                position: 'relative',
+                pr: 3,
+              }}
+            >
+              <AccountLink
+                labelOnly={!!ownAddress && transaction.sender === ownAddress}
+                scope={transaction}
+                address={transaction.sender}
+                alwaysTrim
+              />
+              {verbose && transaction.to && <TransferIcon />}
+            </Box>
           ),
-          key: 'amount',
+          key: 'from',
+        },
+        ...(verbose
+          ? [
+              {
+                content: transaction.to ? (
+                  <AccountLink
+                    labelOnly={!!ownAddress && transaction.to === ownAddress}
+                    scope={transaction}
+                    address={transaction.to}
+                    alwaysTrim
+                  />
+                ) : null,
+                key: 'to',
+              },
+              {
+                align: TableCellAlign.Right,
+                content: getConsensusAmount(t, transaction),
+                key: 'value',
+              },
+            ]
+          : []),
```

Comment:
`verbose` naming is a bit confusing for it, isn't it? Wouldn't it be better to display the columns based on layer? And drop `verbose` flag all together.

Reply:
This matches other code. Could refactor in a next PR, but basing it on layer probably won't be enough in matching code elsewhere